bundles/io/org.eclipse.smarthome.io.transport.dbus/build.properties

```diff
@@ -0,0 +1,9 @@
+bin.includes = META-INF/,\
+               OSGI-INF/,\
+               .,\
+               about.html,\
+               lib/libdbus-java-2.7.jar,\
+               lib/unix-0.5.jar

```

Comment:
This is the library that you wanted to replace by your own implementation, right?

What is the libunix-java.so? Is that needed?